packages/proto-encoder/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@osmosis-labs/proto-encoder",
```

Comment:
Actually, sorry, after knowing what this package is for (coming in and running a command to generate a new coding library for Osmosis. aka Telescope just for Osmosis). I think `proto-encoder` is not a great name. Open to suggestions.

Thoghts on just `protos` ? Or is that missing `encoder` that implies that it has encoding/decoding runtime functions as well? In that case, `proto-coders` (coder is generic across encoder and decoder)?

Reply:
I also thought about using `protos`, but I feel it does not give a good idea of what the package contains since it also includes encoders and decoders.  I like `proto-coders` sounds catchy. However, what about `proto-codecs`? Codecs is a general term for encoder and decoder. wdyt?

Reply:
SGTM